void/reducer.py

```diff
@@ -39,30 +39,37 @@ def read_header_data(fits_fname):
         date_obs = header_dict['DATE-OBS']
         exp = header_dict['EXPTIME']
         focus = header_dict['FOCUSPOS']
-        ra_center = header_dict['OBJCTRA']
-        dec_center = header_dict['OBJCTDEC']
+        ra_center = header_dict['CRVAL1']
+        dec_center = header_dict['CRVAL2']
 
         x_pix_size = header_dict['NAXIS1']
         y_pix_size = header_dict['NAXIS2']
-        x_binning = header_dict['XBINNING']
-        y_binning = header_dict['YBINNING']
 
-        x_scale = config.SCALE_X_BIN1
-        y_scale = config.SCALE_Y_BIN1
+        # Scaling factors in [deg/px]
+        x_scale = abs(header_dict['CDELT1'])
+        y_scale = abs(header_dict['CDELT2'])
 
-        x_deg_size = float(x_pix_size * x_binning * x_scale) / 3600
-        y_deg_size = float(y_pix_size * y_binning * y_scale) / 3600
+        x_deg_size = float(x_pix_size * x_scale)
+        y_deg_size = float(y_pix_size * y_scale)
+
+        pos_angle = header_dict['PA']
+
+        # Limiting magnitude
+        # mag_norm = header_dict['starZMAG']
```

Comment:
card name is `ZMAG`, not `starZMAG`, check this:
```
In [1]: from astropy.io import fits                                                                                                                                                                                                                                                                                                                                  

In [2]: f = fits.open('void/tests/data/test_unflagged.fit')                                                                                                                                                                                                                                                                                                          

In [3]: h = f[0].header                                                                                                                                                                                                                                                                                                                                              

In [4]: h['ZMAG']                                                                                                                                                                                                                                                                                                                                                    
Out[4]: 19.577995437

In [5]: h                                                                                                                                                                                                                                                                                                                                                            
Out[5]: 
SIMPLE  =                    T                                                  
BITPIX  =                   16 /8 unsigned int, 16 & 32 int, -32 & -64 real     
NAXIS   =                    2 /number of axes                                  
...                 
RA      = '11 11 13.00'        / [hms J2000] Target right ascension             
DEC     = '+63 18 32.0'        / [dms +N J2000] Target declination              
FWHM    =   2.50253389478E+000 / [pixels] Mean Full-Width-Half-Max of image star
ZMAG    =   1.95779954370E+001 / Mag zero point for 1 sec exposure              
...
```

Do you see the ⭐️? :trollface:

Reply:
nice, got it